app/boards/shields/contra/boards/nice_nano.overlay

```diff
@@ -0,0 +1,28 @@
+&spi1 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	mosi-pin = <45>; // Data cable is on D15 of Nice!NanoV2
+	// Unused pins, needed for SPI definition, but not used by the ws2812 driver itself.
+	sck-pin = <5>;
+	miso-pin = <7>;
+
+	led_strip: ws2812@0 {
+		compatible = "worldsemi,ws2812-spi";
+		label = "SK6812mini";
+
+		/* SPI */
+		reg = <0>; /* ignored, but necessary for SPI bindings */
+		spi-max-frequency = <4000000>;
+
+		/* WS2812 */
+		chain-length = <6>; /* Change this according to how many leds are on the strip */
+		spi-one-frame = <0x70>;
+		spi-zero-frame = <0x40>;
```

Comment:
Color mapping is now required: https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/lily58/boards/nice_nano.overlay#L29
could you add this? Otherwise LGTM

Reply:
@Nicell I added the missing lines for color mapping